tests/system/providers/google/cloud/dataprocgdc/example_dataprocgdc.py

```diff
@@ -0,0 +1,77 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Example Airflow DAG that shows how to create and monitor a Spark Application or App Environment
+ in a DPGDC cluster
+"""
+
+from __future__ import annotations
+
+from datetime import datetime
+
+from airflow import DAG
+from airflow.providers.google.cloud.operators.dataprocgdc import (
+    DataprocGdcCreateAppEnvironmentKrmOperator,
+    DataprocGDCSubmitSparkJobKrmOperator,
+)
+from airflow.providers.google.cloud.sensors.dataprocgdc import DataprocGDCKrmSensor
+
+DAG_ID = "example_dag_dpgdc-krm"
+
+with DAG(
```

Comment:
Did you check this DAG on your local env?

Reply:
Yes.